cli/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MPL-2.0",
   "dependencies": {
     "caporal": "^1.3.0",
+    "nodegit": "^0.26.2",
     "ssr": "0.1.0"
```

Comment:
No ignorance detected, that's what it does! `nodegit` is nodejs bindings for libgit, I opted for doing that instead of running bash git commands, as I find it more expressive, less error-prone and more cross-platformy.
I'm not sure if that answers the question though 🙃

Reply:
I wasn't referring to `nodegit`, I was referring to `ssr@0.1.0` which is the other local yarn workspace. It's not the name of a package on npmjs. Which, confusingly, could be [a thing](https://www.npmjs.com/package/ssr).

Reply:
Oh right, that makes more sense.

That's how you declare dependencies within a yarn (or lerna) workspace. It's basically like [`yarn link`](https://yarnpkg.com/lang/en/docs/cli/link/) but more convenient.